From a113facc93033cf4c439539a7166b9d33ee81470 Mon Sep 17 00:00:00 2001
From: Oliver Brown <oliver.brown@freescale.com>
Date: Tue, 7 Feb 2012 11:20:02 -0600
Subject: [PATCH 234/635] ENGR00172675: Speaker not controlled by headphone
 jack detect.

Removed the default speaker control for SGTL5000. The speaker
is control by the kernel.

Signed-off-by: Oliver Brown <oliver.brown@freescale.com>
---
 alsa/alsa_imx.cpp | 13 -------------
 1 file changed, 13 deletions(-)

diff --git a/alsa/alsa_imx.cpp b/alsa/alsa_imx.cpp
index 631b40c..a54d0be 100755
--- a/alsa/alsa_imx.cpp
+++ b/alsa/alsa_imx.cpp
@@ -645,19 +645,6 @@ void setDefaultControls(uint32_t devices, int mode, const char *cardname)
     ALSAControl *ctl = new ALSAControl(cardname);
     LOGD ("setDefaultControls set card: %s",cardname);
 
-    if(devices & IMX_OUT_CODEC_DEFAULT)
-    {
-        if(selecteddevice == DEVICE_SGTL5000)
-        {
-            if (devices & AudioSystem::DEVICE_OUT_SPEAKER ||
-                devices & AudioSystem::DEVICE_OUT_EARPIECE) {
-                ctl->set("Speaker Function", "on"); // on
-            } else {
-                ctl->set("Speaker Function", "off"); // off
-            }
-        }
-    }
-
     if(devices & IMX_IN_CODEC_DEFAULT)
     {
         if(selecteddevice == DEVICE_SGTL5000)
-- 
1.8.0

